internal/templatefn/fn.go

```diff
@@ -42,6 +42,7 @@ func All(globals *Globals) map[string]any {
 		"isEnglish":          isEnglish,
 		"isWelsh":            isWelsh,
 		"input":              input,
+		"button":             button,
```

Comment:
I'm calling it button because we might want to do a `{{ template "button" (button ...) }}` later